march_hardware_interface/include/march_hardware_interface/march_hardware_interface.h

```diff
@@ -59,11 +59,13 @@ class MarchHardwareInterface : public march_hardware_interface::MarchHardware {
   bool hasPowerDistributionBoard;
   boost::shared_ptr<controller_manager::ControllerManager> controller_manager_;
   double p_error_, v_error_, e_error_;
+  SoftJointLimits soft_limits_;
```

Comment:
Aren't these already created and added to a handle?

Reply:
yes, but the handle has no get limits